File: arena-core/src/main/java/org/uqbar/arena/graphics/Image.java
package org.uqbar.arena.graphics;

/**
 * @author jfernandes
 */
public class Image {
	private String pathToFile;

	public Image(String pathToFile) {
		this.pathToFile = pathToFile;
	}

	public String getPathToFile() {
		return pathToFile;
	}

	public void setPathToFile(String pathToFile) {
		this.pathToFile = pathToFile;
	}

}

File: arena-core/src/main/scala/org/uqbar/arena/graphics/Image.scala
package org.uqbar.arena.graphics

/**
 * @author jfernandes
 */
class Image(var pathToFile: String) {
	var scale : (Int,Int) = _
	
	// just to be used from java
	def size(width:Int, height:Int) {
	  scale = (width, height)
	}
}